examples/samplers/fast_sampling_with_jax_and_numba.ipynb

```diff
@@ -19,18 +19,58 @@
    "cell_type": "markdown",
```

Comment:
<p>You should set <code>xla_force_num_host_devices=8</code> to make the comparison fair. You're sampling sequentially in this example.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I thought <code>chain_method="parallel"</code> was the default? </p>

Reply:
OK, I've added `numpyro.set_host_device_count(8)` which seems to do the trick.

Reply:
I also thought it was the default, but apparently not.

Internally, `numpyro.set_host_device_count(8)` just does `os.environ["XLA_FLAGS"] = '--xla_force_host_platform_device_count=8'`. My preference is that we show setting that flag directly, rather than using the numpyro function. For a long time I thought that numpyro function did something special, so I was importing that whole package even when I was sampling with nutpie.